.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -0,0 +1,101 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: yaml -*-
+
+jobs:
+- job: win
+  pool:
+    vmImage: vs2017-win2016
+  timeoutInMinutes: 360
+  strategy:
+    maxParallel: 4
+    matrix:
+      win_cxx_compilervs2015vc14:
+        CONFIG: win_cxx_compilervs2015vc14
+        CONDA_BLD_PATH: D:\\bld\\
```

Comment:
Can we change this to `C:`? When [building `qgis`](https://github.com/conda-forge/qgis-feedstock/pull/45#issuecomment-473987293) this seemed to give more drive space (the lack of which seems to be the reason why the build is currently failing). I'm not sure if there are any issues with doing this though...